src/VideoEmbed.php

```diff
@@ -81,4 +82,24 @@ function (PluginEvent $event) {
     // Protected Methods
     // =========================================================================
 
+    /**
+     * @return \craft\base\Model|null
+     */
+    protected function createSettingsModel()
+    {
+        return new \viget\videoembed\models\Settings();
```

Comment:
You should be able to change this to `return new Settings();` since you've got line 14 above